docs/sidebars.js

```diff
@@ -1,54 +1,65 @@
 module.exports = {
-  examples: [
-    "examples/introduction",
+  guides: [
+    "guides/overview",
     {
       type: "category",
-      label: "Integrate ZITADEL Login in your App",
+      label: "Get started",
+      collapsed: false,
       items: [
-        "examples/login/angular",
-        "examples/login/react",
-        "examples/login/flutter",
-        "examples/login/nextjs",
-        "examples/login/nextjs-b2b",
+        "guides/start/quickstart",
       ],
-      collapsed: false,
-    },
-    {
-      type: "category",
-      label: "Secure your API",
-      items: ["examples/secure-api/go", "examples/secure-api/python-flask", "examples/secure-api/dot-net"],
-      collapsed: false,
     },
+    "examples/sdks",
```

Comment:
I think we should move this below quickstarts or into integrate

Reply:
Yes makes sense and pulls together the get started section(s)